orm/go.mod

```diff
@@ -5,26 +5,33 @@ go 1.17
 require (
 	github.com/cosmos/cosmos-proto v1.0.0-alpha7
 	github.com/cosmos/cosmos-sdk/api v0.1.0-alpha5
-	github.com/cosmos/cosmos-sdk/errors v1.0.0-beta.3
+	github.com/cosmos/cosmos-sdk/errors v1.0.0-beta.4
 	github.com/golang/mock v1.6.0
 	github.com/google/go-cmp v0.5.7
 	github.com/iancoleman/strcase v0.2.0
+	github.com/regen-network/gocuke v0.6.1
```

Comment:
Please do not add this dependency.

Reply:
You already expressed your opinion above quite clearly. No need to state this again. It is not helpful. Closing this